dist/api_data.py

```diff
@@ -183,6 +183,12 @@ def __cmp__(self, other):
         configure Huffman encoding for values.  Permitted values are
         \c "none", \c "english", \c "utf8<file>" or \c "utf16<file>".
         See @ref huffman for more information'''),
+    Config('in_memory_ignore_cache_size', 'false', r'''
+        allow update and insert operations to proceed even if the cache is
+        already at capacity. Only valid in conjunction with in-memory
+        databases. Should be used with caution - this configuration allows
+        WiredTiger to consume memory over the configured cache limit''',
+        type='boolean'),
```

Comment:
How about renaming this `ignore_in_memory_cache_size`?  I don't feel super strongly, but having the verb at the beginning reads better to me.